substrate/frame/contracts/Cargo.toml

```diff
@@ -44,7 +44,7 @@ frame-benchmarking = { path = "../benchmarking", default-features = false, optio
 frame-support = { path = "../support", default-features = false }
 frame-system = { path = "../system", default-features = false }
 pallet-balances = { path = "../balances", default-features = false, optional = true }
-pallet-contracts-uapi = { path = "uapi" }
+pallet-contracts-uapi = { path = "uapi", default-features = false, features = ["scale"] }
```

Comment:
This crate is `no_std` only. It doesn't even have a `std` feature. Do you still want to add the `default-features = false`.

Reply:
Oh right that crate is nostd-only, thanks!